src/backend/spies/timeSpy.ts

```diff
@@ -74,6 +76,31 @@ export class TimeSpy {
         return 1000 * 60 / accumulator;
     }
 
+    public listenXRSession(session: XRSession) {
+        if (this.currentXRSession) {
+            this.unlistenXRSession();
+        }
+        for (const Spy of TimeSpy.requestAnimationFrameFunctions) {
+            this.unspyRequestAnimationFrame(Spy, this.spiedWindow);
```

Comment:
1. Yes, we unspy RAF because window RAF will collide with XRSession RAF and cause things to break; I don't think any other timers cause that problem.
2. I can make a new XR Spy module to deal with this and the XR injection.